Makefile

```diff
@@ -83,7 +83,15 @@ update_assets: update_npm_deps
 	$(DOCKER_PREFIX) python manage.py compress_assets
 	$(DOCKER_PREFIX) python manage.py collectstatic --noinput
 
-initialize_docker: initialize_db update_assets
+initialize_docker:
+	$(DOCKER_PREFIX) python manage.py reset_db --noinput
```

Comment:
These don't need `DOCKER_PREFIX`, since this should always be run directly inside the container. And I'd really like to get rid of `DOCKER_PREFIX` altogether.